src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Stream.DeserializeAsyncEnumerable.cs

```diff
@@ -7,6 +7,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
+using Utf8MemoryStream = System.Text.Json.Tests.Serialization.CollectionTests.Utf8MemoryStream;
```

Comment:
Why not import the namespace `System.Text.Json.Tests.Serialization.CollectionTests`?

Reply:
I wanted to avoid bringing in lots of unrelated types into scope.